kpatch-build/create-klp-module.c

```diff
@@ -502,6 +535,8 @@ int main(int argc, char *argv[])
 	kpatch_create_shstrtab(kelf);
 	kpatch_create_strtab(kelf);
 	kpatch_create_symtab(kelf);
+	kpatch_dump_kelf(kelf);
```

Comment:
Needed for future debug?

Reply:
Yes. I wanted to print the final dump coming out of create-klp-module. For debugging, It was beneficial for me.